nodejs/src/mcp-client.js

```
@@ -0,0 +1,685 @@
+      return this.parseToolResponse(result);
+    } catch (error) {
+      throw new Error(`Failed to get session: ${error.message}`);
+    }
+  }
+
+  // Helper method to parse tool response in different formats (for JSON responses)
+  parseToolResponse(result) {
+    if (typeof result === 'string') {
+      return JSON.parse(result);
+    } else if (result.content && result.content[0] && result.content[0].text) {
+      return JSON.parse(result.content[0].text);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)